src/App.tsx

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
-import './App.css';
+import Header from './components/Header';
+import Aside from './components/Aside';
+import CalendarMain from './components/CalendarMain';
+import { useState } from 'react';
 
 function App() {
+  const [currentDate, setCurrentDate] = useState(new Date());
+  const handleDateChange = (newDate: Date) => {
+    setCurrentDate(newDate);
+  };
   return (
-    <div className="App">
-        <h1>ok</h1>
+    <div className="app-container">
```

Comment:
the convention in react is usually to have component and its styles in same folder close by. so App.tsx and app.css (with app-container) styles should be in src, Aside.tsx and aside.css should be together in its own folder etc.

Talking about styling, you can also read about css modules, since styling components using this syntax `className={styles.app}` is very popular.
Also there is awesome package we use called [classnames](https://www.npmjs.com/package/classnames), that helps with class concatenation, conditional classes and similar.